Doc/library/crypt.rst

```diff
@@ -20,6 +20,14 @@
    The :mod:`crypt` module is deprecated
    (see :pep:`PEP 594 <594#crypt>` for details and alternatives).
    The :mod:`hashlib` module is a potential replacement for certain use cases.
+   The :mod:`passlib` package can replace all use cases of this module.
```

Comment:
You need to use a ReST hyperlink to the PyPI [passlib](https://pypi.org/project/passlib/) package.

We should not be including examples of third party software use in our documentation.  At most links to some specific external packages as a suggested replacement that people could look into.

Reply:
Fixed